### PR TITLE
Remove unused nisei env vars.

### PR DESCRIPTION
**Pull request by plural**
tidying up a bit.

### CODE REVIEW

File: deploy/bin/github-actions-set-droplet-env [hunk 1]
```diff
@@ -30,8 +30,6 @@ chmod u=rw,g=,o= "$ENV_FILE"
   echo "COBRA_DOMAIN=$(pulumi stack output cobra_domain)"
   echo "NRDB_CLIENT=$(pulumi stack output nrdb_client)"
   echo "NRDB_SECRET=$(pulumi stack output nrdb_secret --show-secrets)"
-  echo "NISEI_NRDB_CLIENT=$(pulumi stack output nisei_nrdb_client)"
-  echo "NISEI_NRDB_SECRET=$(pulumi stack output nisei_nrdb_secret --show-secrets)"
```
Comment (haradan) [inline deploy/bin/github-actions-set-droplet-env:34]: These need removing from the Pulumi configuration as well, although I could do that in a separate PR if you want.

**Review by haradan — APPROVED** [review]

File: deploy/bin/github-actions-set-droplet-env (hunk 1 above)
Comment (plural) [reply deploy/bin/github-actions-set-droplet-env:34]: Done.
